source/_integrations/teslemetry.markdown

```diff
@@ -13,82 +14,98 @@ ha_domain: teslemetry
 ha_platforms:
   - climate
   - diagnostics
+  - media_player
   - sensor
 ha_integration_type: integration
 ---
 
-The Teslemetry integration exposes various commands and sensors from the Tesla vehicles connected to a [Teslemetry](https://teslemetry.com/) subscription.
+The Teslemetry integration exposes various commands and sensors from the Tesla vehicles and energy sites connected to a [Teslemetry](https://teslemetry.com/) subscription.
 
 ## Prerequisites
 
-You must have a [Teslemetry](https://teslemetry.com/) account and [access token](https://teslemetry.com/console).
+You must have a [Teslemetry](https://teslemetry.com) account, active subscription, and [access token](https://teslemetry.com/console).
+
+Vehicles delivered in 2024 and later will require a [virtual key](https://teslemetry.com/docs/topics/virtualkey) to be configured in order to run certain commands.
 
 {% include integrations/config_flow.md %}
 
 ## Entities
 
-### Climate
-
-The integration will create a climate entity to control the vehicle's climate control system. This entity can:
+These are the entities available in the Teslemetry integration. Not all entities are enabled by default, and not all values are always available.
 
-- Turn on and off
-- Change the set temperature
-- Change to one of the four modes: Off, Keep mode, Dog mode, and Camp mode 
+### Vehicles
 
-### Sensor
+|Domain|Name|Enabled|
+|---|---|---|
+|Climate|Cabin overheat protection|Yes|
+|Climate|Climate|Yes|
+|Media player|Media Player|Yes|
```

Comment:
```suggestion
|Media player|Media player|Yes|
```
Was the name spelled in heading-style capitalization on purpose?

Reply:
It was consistent, but incorrect, so I've updated both the documentation and the integration.